mediainfojs.cpp

```diff
@@ -18,15 +18,29 @@ class MediaInfoJs {
   int open_buffer_continue(const std::string& data, double size) {
     return mi.Open_Buffer_Continue((ZenLib::int8u*)data.data(), (ZenLib::int64u)size);
   }
-  long open_buffer_continue_goto_get(){
-    return (int64_t)mi.Open_Buffer_Continue_GoTo_Get();
+  int open_buffer_continue_goto_get(){
+    seekTo.total = mi.Open_Buffer_Continue_GoTo_Get();
+    return seekTo.lower;
+  }
+  int get_seek_to_upper() {
+    return seekTo.upper;
   }
   MediaInfoLib::String inform() {
     return mi.Inform();
   }
   void close() {
     mi.Close();
   }
+private:
+  union SeekTo {
+    int64_t total;
+    struct {
+      int32_t lower;
+      int32_t upper;
```

Comment:
I recommend `low` and `high`, more classic (and actually you use these words in the other PR about doc).

Reply:
Fair enough. I will change that.